docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-is-used-on-the-project-level.md

```diff
@@ -14,11 +14,17 @@ related:
     link: docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-method-is-overridden-on-the-project-level.html
 ---
 
+Private API is used on the project level. The page describes why entities on the project side have to avoid private api using. The project has to follow the rule to be able to correctly receive Spryker autoupdates.
+
 Modules have public and private APIs. While public API updates always support backward compatibility, private API updates can break backward compatibility. So, backward compatibility is not guaranteed in the private API. For example, if you use a core method on the project level, and it is updated or removed with an update, it can cause unexpected issues.
 
 For more information about module APIs, see [Definition of Module API](/docs/scos/dev/architecture/module-api/definition-of-module-api.html).
 
-## Example of code that causes an upgradability error
+## PrivateApi:Facade
+
+Facade on project level can use repository/factory/entityManager but only methods that were declared on project level.
```

Comment:
It should be better formulated, we can't leave it like that. I think about how to formulate it better

Reply:
++

Replaced with the suggested structure